build.js

```diff
@@ -76,13 +76,25 @@ return Promise.resolve()
         '@angular/common': 'ng.common',
         '@angular/core': 'ng.core',
         '@angular/forms': 'ng.forms',
+        '@angular/http': 'ng.http',
         '@angular/platform-browser': 'ng.platformBrowser',
         '@angular/router': 'ng.router',
         'rxjs/Observable': 'Rx',
         'rxjs/Subscription': 'Rx',
         'rxjs/Subject': 'Rx',
         'rxjs/ReplaySubject': 'Rx',
-        'rxjs/add/observable/skip': 'Rx.Observable',
+        'rxjs/add/observable/from': 'Rx.Observable',
```

Comment:
Sort of curious what would happen if the parent app hasn't already imported these addons.  I guess we'll tackle that another day.